community/portworx/templates/portworx-stork.yaml

```
@@ -61,10 +61,9 @@ kind: ClusterRole
 apiVersion: {{ template "rbac.apiVersion" . }}
 metadata:
    name: stork-role
```

This should be a two space indent, I think.
```suggestion
  name: stork-role
```